mlflow/langchain/__init__.py

```diff
@@ -1063,6 +1063,13 @@ def autolog(
             MlflowLangchainTracer as a callback during inference. If ``False``, no traces are
             collected during inference. Default to ``True``.
     """
+    if log_inputs_outputs is not None:
```

Comment:
We have this warning in AutoLoggingConfig.init, can we remove this, otherwise it warns twice

Reply:
ooh thanks for the catch!